engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1270,7 +1269,7 @@ static void LogFrameBufferError(GLenum status)
             OpenGLIsExtensionSupported(context, "OES_texture_compression_astc") ||
             OpenGLIsExtensionSupported(context, "WEBGL_compressed_texture_astc"))
         {
-            context->m_TextureFormatSupport |= 1 << TEXTURE_FORMAT_RGBA_ASTC_4x4;
+            context->m_ASTCSupport = 1;
```

Comment:
No need to set specific formats here, they should all be supported if we have any of the extensions.